api/caching/tasks.py

```diff
@@ -104,11 +104,20 @@ def ban_url(instance):
 
 @app.task(max_retries=5, default_retry_delay=10)
 def update_storage_usage_cache(target_id):
-    AbstractNode = apps.get_model('osf.AbstractNode')
-
-    storage_usage_total = AbstractNode.objects.get(
-        guids___id=target_id,
-    ).files.aggregate(sum=models.Sum('versions__size'))['sum'] or 0
+    sql = """
+                SELECT sum(version.size) FROM osf_basefileversionsthrough AS obfnv
+                LEFT JOIN osf_basefilenode file ON obfnv.basefilenode_id = file.id
+                LEFT JOIN osf_fileversion version ON obfnv.fileversion_id = version.id
+                LEFT JOIN osf_guid guid ON file.target_object_id = guid.object_id
+                WHERE file.provider = 'osfstorage' AND file.target_content_type_id = 4
```

Comment:
We should get a little comment here that details what content type == 4 or make it explicit that the target is nodes some other way.

Reply:
Ended up joining with django_content_type. Made more sense this way.